.gitlab/templates/lambda-extension.yaml.tpl

```diff
@@ -48,6 +55,58 @@ build bottlecap ({{ $architecture.name }}):
   script:
     - .gitlab/scripts/build_bottlecap.sh
 
+# Alpine
+build go agent ({{ $architecture.name }}, alpine):
+  stage: build
+  image: registry.ddbuild.io/images/docker:20.10
+  tags: ["arch:amd64"]
+  artifacts:
+    expire_in: 1 hr
+    paths:
+      - .layers/datadog_extension-{{ $architecture.name }}-alpine.zip
+      - .layers/datadog_extension-{{ $architecture.name }}-alpine/*
+  variables:
+    ARCHITECTURE: {{ $architecture.name }}
+    ALPINE: 1
+  script:
+    - cd .. && git clone -b $AGENT_BRANCH --single-branch https://github.com/DataDog/datadog-agent.git && cd datadog-lambda-extension
+    - .gitlab/scripts/build_go_agent.sh
+
+build bottlecap ({{ $architecture.name }}, alpine):
+  stage: build
+  image: registry.ddbuild.io/images/docker:20.10
+  tags: ["arch:amd64"]
+  needs:
+    - build go agent ({{ $architecture.name }}, alpine)
+  dependencies:
+    - build go agent ({{ $architecture.name }}, alpine)
+  artifacts:
+    expire_in: 1 hr
+    paths:
+      - .layers/datadog_bottlecap-{{ $architecture.name }}-alpine.zip
+      - .layers/datadog_bottlecap-{{ $architecture.name }}-alpine/*
+  variables:
+    ARCHITECTURE: {{ $architecture.name }}
+    ALPINE: 1
+  script:
+    - .gitlab/scripts/build_bottlecap.sh
+
+build image ({{ $architecture.name }}, alpine):
+  stage: build
+  tags: ["arch:amd64"]
+  image: registry.ddbuild.io/images/docker:20.10
+  when: manual
+  needs:
+    - build bottlecap ({{ $architecture.name }}, alpine)
+  dependencies:
+    - build bottlecap ({{ $architecture.name }}, alpine)
+  variables:
+    ARCHITECTURE: {{ $architecture.name }}
+    ALPINE: 1
+  script:
+    - .gitlab/scripts/build_image.sh
+# Alpine end
+
```

Comment:
This is better than adding a new entry into `architectures.yaml`, way easier even.